File: BAEKJOON/1924.py
m = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31 ] 
y = [ "SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"]

dal, il = map(int, input().split())
# ex input 3 3
day = 0
for i in range(0,dal-1):
    day+=m[i]

day += il

print(y[day % 7])

File: JAVA/BOJ/1924.java
import java.util.*;

class Main {
  public static void main(String[] args) {
    int[] mondays = {31,28,31,30,31,30,31,31,30,31,30,31};
    String[] week = {"SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"};

    Scanner scanner = new Scanner(System.in);

    int month = scanner.nextInt();
    int day = scanner.nextInt();

    int daysum = 0;
    
    for (int k = 0; k < month - 1 ; k++){
      daysum += mondays[k];
    }

    daysum += day;

    String weekday = (week[daysum % 7]);
    
    System.out.println(weekday);
    
  }
}